portality/app.py

```diff
@@ -62,12 +56,21 @@
 app.register_blueprint(editor, url_prefix='/editor') # ~~-> Editor:Blueprint~~
 app.register_blueprint(services, url_prefix='/service') # ~~-> Services:Blueprint~~
 if 'api1' in app.config['FEATURES']:
+    from portality.view.api_v1 import blueprint as api_v1
     app.register_blueprint(api_v1, url_prefix='/api/v1') # ~~-> APIv1:Blueprint~~
 if 'api2' in app.config['FEATURES']:
+    from portality.view.api_v2 import blueprint as api_v2
     app.register_blueprint(api_v2, url_prefix='/api/v2') # ~~-> APIv2:Blueprint~~
 if 'api3' in app.config['FEATURES']:
-    app.register_blueprint(api_v3, name='api', url_prefix='/api') # ~~-> APIv3:Blueprint~~
+    from portality.view.api_v3 import blueprint as api_v3
     app.register_blueprint(api_v3, name='api_v3', url_prefix='/api/v3') # ~~-> APIv3:Blueprint~~
+if 'api4' in app.config['FEATURES']:
+    from portality.view.api_v4 import blueprint as api_v4
+    app.register_blueprint(api_v4, name='api_v4', url_prefix='/api/v4') # ~~-> APIv3:Blueprint~~
+if 'api_current' in app.config['FEATURES']:
+    from portality.view.api_current import blueprint as api_current
+    app.register_blueprint(api_current, name='api', url_prefix='/api') # ~~-> APICurrent:Blueprint~~
```

Comment:
api_current and api_v4 are the same, so there's no need for an `api_current` blueprint, we can just bind `api_v4` to both url routes, that works fine.